cypress/integration/Page/publicAuctionFilterPage.js

```diff
@@ -107,15 +53,8 @@ describe('Testcase for Manheim filter', () => {
                 })
             })
         })
-    })
-    it('Verify owner filter', () => {
-        cy.visit('/')
-        cy.get('.uhf-user__locations').click()
-        cy.contains('U.S. Locations').click()
-        cy.contains('Manheim Cincinnati').click()
-        cy.contains('Public Auctions').click()
-        cy.contains('See Inventory').click()
-        cy.wait(10000)
+    }
+    verifyOwner(){
         cy.get('[data-test-id="filter: owners"]').then(owner => {
             cy.wrap(owner).click()
             cy.get('[data-test-id="AutocheckOwnerCountFilter"]').select('1')
```

Comment:
We shouldn't hardcode at `.select('1')`, because the test case will fail when the Owner filter doesn't have data `1`. We should implement something like this: `cy.get('[data-test-id="AutocheckOwnerCountFilter"]').first.select()`